staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -396,6 +400,11 @@ func (r *crdHandler) serveResource(w http.ResponseWriter, req *http.Request, req
 		forceWatch := true
 		return handlers.ListResource(storage, storage, requestScope, forceWatch, r.minRequestTimeout)
 	case "create":
+		if justCreated {
+			err := apierrors.NewTooManyRequests("not available while waiting for other kube-apiservers to observe", 5)
+			err.ErrStatus.Message = "not available while waiting for other kube-apiservers to observe"
+			responsewriters.ErrorNegotiated(err, Codecs, schema.GroupVersion{Group: requestInfo.APIGroup, Version: requestInfo.APIVersion}, w, req)
+		}
```

Comment:
would you change all wait logic for CRDs in the tests? And if yes, how?

Reply:
> would you change all wait logic for CRDs in the tests? And if yes, how?

our client retries on 429.  I don't think they need to update.  If they do, this approach is bad.

Reply:
kubectl doesn't. why would we fail this instead of holding the request for 2 seconds?

Reply:
switched to sleep